packages/ui/src/working-groups/components/WorkingGroupsList.tsx

```diff
@@ -34,14 +36,18 @@ export const WorkingGroupsList = ({ groups }: WorkingGroupsListProps) => {
 }
 
 const HeaderColumnWrapper = styled.div`
-  display: flex;
-  justify-content: flex-end;
+  display: grid;
+  grid-template-columns: 108px repeat(2, minmax(400px, 1fr)) 40px;
```

Comment:
Please export the value of the `GroupItem` `grid-template-columns` from  `WorkingGroupListItem.tsx` and re-use it here